### PR TITLE
COMMERCE-11647

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-11379](https://issues.liferay.com/browse/COMMERCE-11379)

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSkusVirtualSettingsOverride.testcase [hunk 1]
```diff
@@ -37,6 +37,152 @@ definition {
...
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceNavigator.gotoTableListEntry(entryName = "VIRTUALSKU");
+
+			CommerceProducts.updateProductVirtualSettingOverride(
+				activationStatus = "Pending",
+				key_contentType = "Base Information",
+				key_fieldLabel = "Activation Status",
+				maxDownloads = 3,
+				virtualProductFile = "Commerce_Black.jpg");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSkusVirtualSettingsOverride.testcase:79]: @Brunno0liveira to make sure that the override is actually changing the file, can you change the virtualProductFile to "Commerce_White.jpg"? Thanks

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:foward

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**